CryptoTechReminderSystem.AcceptanceTest/CryptoTechReminderSystemTests.cs

```diff
@@ -23,6 +23,10 @@ public class CryptoTechReminderSystemTests
         private static HarvestGateway _harvestGateway;
         private static SlackGateway _slackGateway;
         private static SendReminder _sendReminder;
+        private static readonly string Roles =  
+            "Software Engineer, Senior Software Engineer, Senior Engineer, Lead Engineer, "+
+            "Delivery Manager, SRE, Consultant, Delivery Principal";
+        
```

Comment:
Unnecessary blank line.

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/31/commits/00f4b59914005bca399412f767614ae41feebd50.
Cheers!

Reply:
```suggestion
```